nightfall-client/src/routes/kyc.mjs

```diff
@@ -45,4 +50,23 @@ router.post('/remove', async (req, res, next) => {
   }
 });
 
+/**
+ Validate a certificate (which will also add the user to the whitelist if the certificate is valid and the signature over
+ their ethereum address checks out). The signature can be falsey if we don't want to whitelist the address but are just validating
+ the certificate.  We might want to do this for an intermediate certificate for example.
+ */
+router.post('/validate', async (req, res, next) => {
+  const { certificate, ethereumAddressSignature } = req.body;
+  if (!certificate.type === 'Buffer') next(new Error('Certificate is not a buffer'));
```

Comment:
I don't really follow I'm afraid.  I'd expect an error if you don't sent a certificate so that seems like correct behaviour.

Reply:
Yes, an error that will happen will be something like this `Property type can't be read from certificate`, then adding a validation for this would raise a more explicit message.